ui/desktop/tests/e2e/fixtures/electronTest.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+/* eslint-disable no-undef, ember/avoid-leaking-state-in-ember-objects */
+const { test } = require('@playwright/test');
+const path = require('path');
+const fs = require('fs');
+
+/**
+ * Get the path to the Boundary Desktop executable
+ * @param buildDirectory
+ * @return {string}
+ */
+const getExecutablePath = (buildDirectory = 'out') => {
+  const rootDir = path.resolve('./electron-app');
+  // directory where the builds are stored
+  const outDir = path.resolve(rootDir, buildDirectory);
+  // list of files in the out directory
+  const builds = fs.readdirSync(outDir);
+
+  // list of platforms we support
+  const platforms = ['win32', 'darwin', 'linux'];
```

Comment:
Not blocking comment: Just heads up, once we merge `llb-improvements-and-transparent-session-changes` change within the pipeline, `win32` changes to `windows`.

Context on the change: `win32` is a not accepted by our blobl signature tool and has to be `windows`.

Reply:
Yup good call, I can also add `windows` to the list for now and add a todo to remove `win32` later

Reply:
Actually I'm not sure it matters? I'm looking for the initial build made by electron forge which should be unmodified and not the one that we change in `postMake` hooks, does that sound right? It also looks like we already have the rename in `main`